app/shared/api-services/poll/answer-factory.tests.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+describe('Answer Factory', function () {
+  var $httpBackend,
+      answer,
+      answerFactory,
+      scope,
+      pollId;
+
+  beforeEach(module('apiServices.answer.factory'));
+
+  beforeEach(inject(function(_$httpBackend_, _Answer_) {
+    answerFactory = _Answer_;
```

Comment:
its an angular thing so you can inject a service under a different name and assign a local variable of the same name as the service. It's supposed to make your tests more readable. Although, it's unnecessary in the case of the answerFactory variable.